ui/src/app/utils/breakLines.ts

```diff
@@ -3,11 +3,15 @@
  * that is inserted into a <pre> block.
  */
 export const breakLines = (
-  text: string,
+  text: string | null,
   breakAtSpaces = true,
   lineLength = 52
 ): string => {
   let chunks = [];
+
+  if (text == null) {
```

Comment:
=== ;)

Reply:
I specifically wanted to target both null and undefined in case this ever comes as undefined from the API. Can change to `===`.